res/css/views/elements/_SSOButtons.scss

```diff
@@ -45,6 +45,10 @@ limitations under the License.
         }
     }
 
+   .mx_SSOButton:hover {
+        background-color: #F5F5F5;
```

Comment:
this ought to really be a theme variable, this colour doesn't appear to be used anywhere else in the app so is far from great.

Reply:
Switched to $panel-hover which seems to be in the acceptable to use group of variables <img width="224" src="https://user-images.githubusercontent.com/5855073/148772548-da7163f0-817b-4484-8bfe-d5c7aada7180.png">